### ISSUE TITLE
ERROR: user: Current not implemented on linux/amd64

### ISSUE THREAD

**Issue by NuxRo** [alexzorin/onapp#3]
I downloaded the latest release and tried to run it only to get the error in the subject.
Any idea?

**Comment by alexzorin** [comment]
Hmm. I'm pretty sure if you compile from source with Go 1.7+ it
won't happen.


I haven't updated the releases in a while. To be honest the project is a
bit of an abandoned baby, I still use it but have since rewritten the
underlying library at serversaurus/onapp .




On Thu, Jan 12, 2017, at 12:05 AM, NuxRo wrote:

> I downloaded the latest release and tried to run it only to get the
> error in the subject.
>  Any idea?



> — You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub[1], or mute the
> thread[2].
> 




Links:

  1. https://github.com/alexzorin/onapp/issues/3

**Comment by NuxRo** [comment]
Thanks, I'll try to compile from source.
Is there another cli alternative you'd recommend?

**Comment by alexzorin** [comment]
As far as I am aware there is not another pure-OnApp CLI.

It is possible there are some CLIs for libcloud (which has an OnApp driver) that you can use.

**Comment by riconnon** [comment]
@alexzorin you mention a rewrite of the client library.
Is this available somewhere?

**Comment by alexzorin** [comment]
No, sorry. I had code-generated Go bindings based on the official OnApp
PHP wrapper, but it turns out that the wrapper has many bugs and
inaccuracies in it, such that the generated  statically typed bindings
were pretty worthless in practice.
I'm no longer involved with any OnApp work so this project is
definitely dead.
Alex


On Fri, Oct 6, 2017, at 10:59 PM, irconan wrote:
> @alexzorin[1] you mention a rewrite of the client library.
>  Is this available somewhere?


> — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub[2], or mute the thread[3].> 


Links:

  1. https://github.com/alexzorin
  2. https://github.com/alexzorin/onapp/issues/3#issuecomment-334735069
